charts/testkube-logs-service/values.yaml

```diff
@@ -0,0 +1,197 @@
+# Default values for testkube-cloud-api.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  # -- Domain under which to create Ingress rules
+  domain: ""
+  # -- UI subdomain which get prepended to the domain
+  uiSubdomain: "cloud"
```

Comment:
these global values are not needed here: `uiSubdomain`, `restApiSubdomain`, `grpcApiSubdomain`, `websocketApiSubdomain` as they are not used in the chart.

Reply:
grpcApiSubdomain is used removed rest